exporter/otlpexporter/otlp.go

```diff
@@ -209,3 +233,21 @@ func getThrottleDuration(t *errdetails.RetryInfo) time.Duration {
 	}
 	return 0
 }
+
+// A component status of PermanentError indicates the component is in a state that will require user
+// intervention to fix. Typically this is a misconfiguration detected at runtime. A component
+// PermanentError has different semantics than a consumererror. For more information, see:
+// https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-status.md
+func isComponentPermanentError(st *status.Status) bool {
+	switch st.Code() {
+	case codes.NotFound:
+		return true
```

Comment:
This can easily be resolved correct? If the server is temporary down this should not mark the component as permanent error.